test/e2e/specs/specs-playwright/wp-widgets__spec.ts

```diff
@@ -0,0 +1,50 @@
+import { DataHelper, LoginFlow, SidebarComponent, setupHooks } from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+const gutenbergUser =
+	process.env.GUTENBERG_EDGE === 'true' ? 'gutenbergSimpleSiteEdgeUser' : 'gutenbergSimpleSiteUser';
+
+describe( DataHelper.createSuiteTitle( 'Widgets' ), function () {
+	let sidebarComponent: SidebarComponent;
+	let page: Page;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	it( 'Log in', async function () {
+		const loginFlow = new LoginFlow( page, gutenbergUser );
+		await loginFlow.logIn();
+	} );
+
+	// @todo: Refactor/Abstract these steps into a WidgetsEditor component
```

Comment:
I can help with this part once you merge this PR :)

Reply:
🎉